frontend/components/ImagesSection.js

```diff
@@ -0,0 +1,35 @@
+import { View, Image } from 'react-native'
+import React, { useEffect, useState } from 'react'
+import ImagesCarousel from './ImagesCarousel'
+import useAPI from '../hooks/useAPI'
+
+export default function ImagesSection({ note_id }) {
+    const [images, setImages] = useState([]);
+    const { loading, fetchData } = useAPI(`notes/getimages/${note_id}`, ({ images }) => setImages(images), console.log);
+
+    useEffect(() => {
+        fetchData();
```

Comment:
we run function that makes request. if the request is successful - we run onSuccess else we run onError